lib/rqrcode/export/ansi.rb

```diff
@@ -19,23 +21,20 @@ def as_ansi(options={})
           quiet_zone_size: 4
         }.merge(options)
 
-        normal = "\033[m"
+        normal = "\033[m\n"
         light = options.fetch(:light)
         dark = options.fetch(:dark)
         fill_character = options.fetch(:fill_character)
         quiet_zone_size = options.fetch(:quiet_zone_size)
+        output = []
 
-        output = ''
-
-        @modules.each_index do |c|
-
+        @qrcode.modules.each_index do |c|
```

Comment:
Could this not just be `modules.each_index` now or am I misunderstanding how Forwardable works?

Reply:
It could be yes, but `modules` is not an interface I wanted going forward, as I'd rather everyone went through `@qrcode` in for their extensions. I'd used `.modules` though in one of the example in the README so it's likely somebody has copied that so rather than break lots of stuff I've added the alias.